### PR TITLE
feat(aurora): update install instrux to include meta helm chart

### PR DESCRIPTION
**Pull request by dogonthehorizon**
Adds the new helm chart install method. Includes caveat for users that already have Argo installed. Includes updated network language for our Helm chart repo for those with network restrictions.

### CODE REVIEW

File: content/en/docs/installation/armory-deployments-for-spinnaker.md [hunk 1]
```diff
@@ -159,6 +163,15 @@ kubectl create ns armory-agent
 
 > The examples on this page assume you are using a namespace called armory-agent for the Agent. Replace the namespace in the examples if you are using a different namespace.
 
+#### Install Argo Rollouts
```
Comment (dogonthehorizon) [inline content/en/docs/installation/armory-deployments-for-spinnaker.md]: This is now in the "manual" section for those that don't want to use Helm.

**Review by stephenatwell — APPROVED** [review]

File: content/en/docs/installation/armory-deployments-for-spinnaker.md (hunk 1 above)
Comment (317brian) [reply content/en/docs/installation/armory-deployments-for-spinnaker.md]: Seems odd to be me that someone would say "nope don't want to use helm for this". I'd be in favor of just ripping it out and saying the Helm chart installs everything you need, including Cloud Agent, Argo Rollouts, and (whatever ele we mght add in the future).

And punting the manual steps to the KB (at least when we go GA). Giving people multiple options, especially when one is manual and error prone is 😬  

Will review the whole thing later today or Monday.

Comment (317brian) [reply content/en/docs/installation/armory-deployments-for-spinnaker.md]: Talked to Stephen about this. There are instances where a customer might already have ArgoRollouts

Comment (317brian) [reply content/en/docs/installation/armory-deployments-for-spinnaker.md]: To close this loop, the mega helm chart will be the recommended path. The manual Agent install instructions are being moved to the KB.

**Review by 317brian — APPROVED** [review]
LGTM. @sh4sh will take care of putting the manual agent install steps into the KB for posterity™

**Pull request merged**